app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarMenuControllerTest.kt

```diff
@@ -562,25 +563,36 @@ class DefaultBrowserToolbarMenuControllerTest {
     }
 
     @Test
-    fun `WHEN sync sign in menu item is pressed AND account is signed out THEN navigate to sync sign in`() = runBlockingTest {
-        val item = ToolbarMenu.Item.SyncAccount(false)
-        val directions = BrowserFragmentDirections.actionGlobalTurnOnSync()
-
+    fun `GIVEN account exists and the user is signed in WHEN sync sign in menu item is pressed THEN navigate to account settings`() = runBlockingTest {
+        val item = ToolbarMenu.Item.SyncAccount(AccountState.AUTHENTICATED)
+        val accountSettingsDirections = BrowserFragmentDirections.actionGlobalAccountSettingsFragment()
         val controller = createController(scope = this, store = browserStore)
+
         controller.handleToolbarItemInteraction(item)
 
-        verify { navController.navigate(directions, null) }
+        verify { navController.navigate(accountSettingsDirections, null) }
     }
 
     @Test
-    fun `WHEN sync sign in menu item is pressed AND account is signed in THEN navigate to sync sign in`() = runBlockingTest {
-        val item = ToolbarMenu.Item.SyncAccount(true)
-        val directions = BrowserFragmentDirections.actionGlobalAccountSettingsFragment()
-
+    fun `GIVEN account exists and the user is not signed in WHEN sync sign in menu item is pressed THEN navigate to account problem fragment`() = runBlockingTest {
+        val item = ToolbarMenu.Item.SyncAccount(AccountState.SIGNED_OUT)
+        val accountProblemDirections = BrowserFragmentDirections.actionGlobalAccountProblemFragment()
         val controller = createController(scope = this, store = browserStore)
+
         controller.handleToolbarItemInteraction(item)
 
-        verify { navController.navigate(directions, null) }
+        verify { navController.navigate(accountProblemDirections, null) }
+    }
+
+    @Test
+    fun `GIVEN account doesn't exist WHEN sync sign in menu item is pressed THEN navigate to sign in`() = runBlockingTest {
```

Comment:
```suggestion
    fun `GIVEN account doesn't exist WHEN sign in to sync menu item is pressed THEN navigate to sign in`() = runBlockingTest {
```

This is about aligning it with the same wording as what we see in the toolbar menu.